### DISCUSSION TITLE
I am facing this issue while using this library with laravel 9 and php 8.1

### DISCUSSION THREAD

**Discussion by manzoor555** [spatie/laravel-ciphersweet#50] [General]
# bug

![image](https://github.com/spatie/laravel-ciphersweet/assets/109200732/78834f5f-f612-45a8-8880-269949cbccd7)

**Comment by manzoor555** [comment]
Can you guys help me get rid against this issue ?

**Comment by manzoor555** [comment]
Here you can see sodium is installed on my system
![image](https://github.com/spatie/laravel-ciphersweet/assets/109200732/d1a6ccf9-3cea-4572-96f7-d48c30392eaf)

**Reply by bdj-Razik** [reply]
hi bro
i have same problem 
do you have solution please
